enemytskova/01.HTML/Task4.html

```diff
@@ -0,0 +1,62 @@
+        <ul>
+            <i>
+    	    <li>Программа на Python — часть 1</li>
+    	    <li>Программа на Python — часть 2</li>
+    	    <li>Программа на Python — часть 3</li>
+            </i>
+    	</ul>   
+    	<h2>Содержание</h2>
+    	<h3>HOW-TO: Программа на Python, Часть 2</h3>
+    	<ul style="margin-bottom: 1.5em">
+            <u>
+    		<li>Списки</li>
+    		<li>Комментарии</li> 
+            </u>
+        </ul>
+        <hr>
+        <p>Впрошлом уроке мы рассмотрели простую программу, которая использовала raw_input<br>
+        для получения ответа пользователя, простые типы переменных и простой цикл «<b>for</b>». В<br>
+        этом уроке мы рассмотрим ещё больше типов переменных и напишем ещё больше<br>
+        программ.</p> 
+        <h2>Списки</h2>
+        <p style="margin-bottom: 1.5em">Давайте рассмотрим другой тип переменных, который называется <i>списки</i>. В других<br>
+        языках программирования они называются <i>массивами</i>. Если мы обратимся к аналогии с<br>
+        коробками для обуви, то массив (список) представляет собой несколько склеенных друг с<br>
+        другом коробок с одинаковыми вещами. Например, в одной коробке лежат вилки, в<br>
+        другой — ножи, в третьей — ложки. Посмотрим на простой список— список месяцев. Мы<br>
+        запрограммируем его так:</p>
+<pre><code>months =
+['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec']</code></pre>
+        <p>Чтобы создать список, мы заключили все значения в квадратные скобки ( <b>'['</b> и <b>']'</b> ). Мы<br>
+        назвали наш список '<i>months</i>'. Чтобы использовать список, мы дадим команду print<br>
+        <b>months[0]</b> или <b>months[1]</b> (которые выведут на экран '<i>Jan</i>' или '<i>Feb</i>'). Помните: счет<br>
+        начинается с нуля. Чтобы найти длину списка, можно использовать такую функцию:</p>
+<pre><code>print len(months)</code></pre>
+        <p>которая вернёт 12.</p>
+        <p style="margin-bottom: 1.5em">Ещё один пример списков — категории в поваренной книге. Например...<p/>
+<pre><code>categories = ['Main dish','Meat','Fish','Soup','Cookies']</code></pre>
+        <p style="margin-bottom: 1.5em">Таким образом, в <b>categories[0]</b> хранится '<i>Main dish</i>', а в <b>categories[4]</b> — '<i>Cookies</i>'. Всё<br>
+        очень просто. Я уверен, вы сможете придумать много других примеров использования<br>
+        списков.</p>
+        <hr>
+        <h2>Комментарии</h2>
+        <p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам.<br>
+        Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят<br>
+        вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ,<br>
+        комментарии станут важными. Также при их помощи вы заставите Python игнорировать<br>
+        какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'.<br>
+        Например:</p>
+        #
```

Comment:
it is code too

Reply:
Good evening! Sorry, I dont`t understand what do you mean) Could you explaine me, please